localnet/with-ibc/start.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+if [[ ! -x "$(which jq)" ]]; then
+  echo "jq (a tool for parsing json in the command line) is required..."
+  echo "https://stedolan.github.io/jq/download/"
+  exit 1
+fi
+
+if [[ ! -x "$(which rly)" ]]; then
+  echo "Relayer is required..."
```

Comment:
```suggestion
  echo "Install relayer."
```

Reply:
Now we dont need to build relayer in local.
we are going to use relayer docker image